test/index.test.js

```diff
@@ -74,6 +74,115 @@ describe('mongodb-download-url', function() {
       verify(done, query,
         'https://fastdl.mongodb.org/linux/mongodb-linux-i686-3.0.7.tgz');
     });
+
+    it('should resolve RHEL 6.2', function(done) {
+      // We support 6.2, and 7.0 community versions
+      var query = {
```

Comment:
can these be `const`s?

Reply:
I'm not sure what our rule of thumb should be on updating the style in modules that aren't completely ours as Compass uses this too?

Reply:
sure let's leave this as ES5 for now, no need to change everything at once